cosigned/base/clusterrolebinding.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
should go in: https://github.com/operate-first/apps/tree/master/cluster-scope/base/rbac.authorization.k8s.io 

and add to https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml